doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+  $ sudo umount /mnt
+
+Step 4: Copy the bootable kernel image to the /boot directory::
+
+  $ sudo cp <path-to-kernel-image-built-in-step1>/bzImage /boot/
+
+
+Note: If you update kernel image (bzImage) and modules on ubuntu of KBL NUC i7, need use scp to copy it from the remote host building machine.
+If you install EBtool Docker on the ubuntu of KBL NUC i7, it will be easy to update kernel image and modules.
+
+Update ACRN hypervisor Image
+****************************
+
+Step 1: Before building the ACRN hypervisor, you need to figure out the I/O address of the serial port, and the PCI BDF addresses of the SATA controller and the USB controllers on KBL NUC i7 board.
+Enter the following command to get the I/O addresses of the serial ports. KBL NUC i7 boards support one serial port ttyS0. Need to connect the serial port to the development host, in order to access the ACRN serial console to switch between pre-launched VMs::
+
+    $dmesg | grep ttyS0
+    [    0.000000] console [ttyS0] enabled
+    [    1.562546] 00:01: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
+    The following command prints detailed information about all PCI buses and devices in the system.
+    $lspci -vv
+    00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
+            Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
+    00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
+            Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
+    00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
+            Subsystem: Intel Corporation Ethernet Connection I219-LM
+    Note: Double check PCI devices BDF defined in the "hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h".
+
+Step 2:Clone the ACRN source code and configure the build options with make menuconfig command::
+
+  $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+  $ cd acrn-hypervisor/hypervisor
+  $ make menuconfig
+  Updates the following configure item:
+  Set ACRN Scenario as "Logical Partition VMs";
+  Set Hypervisor mode as "Partition mode";
+  Set Target board as "nuc7i7bnh"
+  Set Maximum number of VCPUs per VM as "2";
+  Set Maximum number of PCPU as "4";
+  Set Serial IO type as "Legacy";
+  Set Base address of serial PIO region as "0x3f8";
+  Set 2M-aligned Start physical address of the RAM region used by the hypervisor as "0x6a000000";
+  Select "Enable ACPI DMA Remapping tables parsing".
+
+Other items is better to keep the default value.
+
+Step 3: Check or update board specific configuration.
+Double check GRUB loader name whether it is provided in the ``hypervisor/arch/x86/configs/nuc7i7bnh/board.c``. If no, should update it::
+
+  update-grub -v
+  grub-mkconfig (GRUB) 2.02~beta2-36ubuntu3.18
+
+In this case, the GRUB loader name shall have ``GRUB`` prefix;
```

Comment:
I'm not too sure what to do in this step, there is no Grub info in that file at the moment.

Reply:
@gvancuts We need to double check the GRUB boot loader name which should have "GRUB" prefix, otherwise, ACRN hypervisor will fail to boot.

Reply:
Where do you check that? Can you give a more explicit example?

Reply:
@gvancuts ACRN hypervisor checks this on the function "init_vboot_operations", please refer to https://github.com/projectacrn/acrn-hypervisor/blob/master/hypervisor/boot/guest/vboot_wrapper.c

Reply:
Sorry but I'm still completely confused about this step. You ask people to check the board specific configuration in `hypervisor/arch/x86/configs/nuc7i7bnh/board.c`, specifically for the "GRUB loader name" but that file (see [here](https://github.com/projectacrn/acrn-hypervisor/blob/master/hypervisor/arch/x86/configs/nuc7i7bnh/board.c)) has no such parameter. So what are people suppose to check??

Reply:
Thanks for your finding. There are some mistakes in this step. It should be "Step 3: Check GRUB boot loader name." we only need to check the GRUB boot loader name on the ubuntu of target board.

Reply:
fix at doc v2 version